accname/name/shadowdom/basic.html

```diff
@@ -0,0 +1,35 @@
+<!doctype html>
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/resources/testdriver.js"></script>
+<script src="/resources/testdriver-vendor.js"></script>
+<script src="/resources/testdriver-actions.js"></script>
+<script src="/wai-aria/scripts/aria-utils.js"></script>
+
+<label id="label1">
+  <div id="host1"></div>
+</label>
+<button id="labelled1"
+        class="labelled"
+        type="button"
+        aria-labelledby="label1"
+        data-expectedlabel="foo"
+        data-testname="content inside shadow DOM"></button>
+
+<label id="label2">
+  <div id="host2"></div>
+</label>
+<button id="labelled2"
+        class="labelled"
+        type="button"
+        aria-labelledby="label2"
+        data-expectedlabel="bar"
+        data-testname="aria-label inside shadow DOM"></button>
```

Comment:
Assuming the remaining checks pass, this is the last minor change request, I think. Overall it looks good.

The test names are effectively permanent identifiers of the tests, and are used to compare test results over time. As such, I think this label (and perhaps the the others) should be more specific about what they test. In this case, it's not just aria-label, but an aria-labelledby relationship that then delegates (?) its label to the shadow contents aria-label... As we add more tests to the file, the current test names should not risk name collision with a future test added at a later time.

So maybe something like "aria-labelledby reference to nameFrom: shadow contents with aria-label value"?

Reply:
Thanks for the explanation! I didn't realize that these test names would be so important. I've updated them to be more descriptive – please let me know if this looks okay.